jobs/JGLOBAL_SNOW_ENSEMBLE_ANALYSIS

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export DATA=${DATA:-${DATAROOT}/${RUN}esnowanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "esnowanl" -c "base esnowanl"
+
+##############################################
+# Set variables used in the script
+##############################################
+# Ignore possible spelling error (nothing is misspelled)
+# shellcheck disable=SC2153
+GDUMP="gdas"
+GDUMP_ENS="enkf${GDUMP}"
+export GDUMP
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMIN_OBS COMOUT_ATMOS_ANALYSIS COMOUT_CONF
+MEMDIR="ensstat" YMD=${PDY} HH=${cyc} declare_from_tmpl COMOUT_SNOW_ANALYSIS
```

Comment:
These will fail, because there are not template with `COMIN_` or `COMOUT_`.  The templates are `COM_`.
```suggestion
YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
  COMIN_OBS:COM_OBS_TMPL \
  COMOUT_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
  COMOUT_CONF: COM_CONF_TMPL
MEMDIR="ensstat" YMD=${PDY} HH=${cyc} declare_from_tmpl 
 COMOUT_SNOW_ANALYSIS:COM_SNOW_ANALYSIS_TMPL
```
Just a note that `COMOUT_SNOW_ANALYSIS` is not being exported, so it will stay local (i.e. child scripts `exscripts...` will not know this variable.